apps/lfx-one/src/app/modules/votes/components/vote-results-drawer/vote-results-drawer.component.html

```
@@ -79,40 +68,113 @@ <h1 class="text-lg sm:text-xl font-semibold text-slate-900 flex-1 min-w-0 trunca
+              <p class="text-sm text-gray-500">No ranked results recorded yet.</p>
+            </div>
+          }
 
-          <a [href]="pccVotingUrl()" target="_blank" rel="noopener noreferrer" class="inline-block">
-            <lfx-button
-              label="View in Project Control Center"
-              icon="fa-light fa-arrow-up-right-from-square"
-              severity="primary"
-              data-testid="vote-results-pcc-link"></lfx-button>
-          </a>
+          @if (commentResults().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)